engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -206,6 +206,20 @@ class AlertReceiveChannel(IntegrationOptionsMixin, MaintainableObject):
     rate_limited_in_slack_at = models.DateTimeField(null=True, default=None)
     rate_limit_message_task_id = models.CharField(max_length=100, null=True, default=None)
 
+    class Meta:
+        constraints = [
+            # This monstrous constraint ensures that there's at most one active direct paging integration per team
+            # This should work with SQLite, PostgreSQL and MySQL >= 8.0.13
```

Comment:
What will happen if some OSS customer using Mysql <8.0.13? Will migration fail?

Reply:
From [the docs](https://docs.djangoproject.com/en/4.2/ref/models/indexes/#expressions):

> Functional indexes are ignored with MySQL < 8.0.13 and MariaDB as neither supports them.

I'll update the comment to reflect this 👍